packages/flutter/lib/src/gestures/resampler.dart

```diff
@@ -6,6 +6,7 @@ import 'dart:collection';
 
 import 'events.dart';
 
+/// A callback for pointer events.
```

Comment:
Ditto

Reply:
@dreveman or @liyuqian can you provide a better description for this one?